modules/cli/src/test/java/org/apache/ignite/cli/commands/CliCommandsErrorTest.java

```diff
@@ -0,0 +1,152 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli.commands;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import io.micronaut.configuration.picocli.MicronautFactory;
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.ignite.cli.commands.configuration.cluster.ClusterConfigShowSubCommand;
+import org.apache.ignite.cli.commands.configuration.cluster.ClusterConfigUpdateSubCommand;
+import org.apache.ignite.cli.commands.configuration.node.NodeConfigShowSubCommand;
+import org.apache.ignite.cli.commands.configuration.node.NodeConfigUpdateSubCommand;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import picocli.CommandLine;
+
+/**
+ * Base class for testing error handling in CLI commands that use REST.
+ */
+@MicronautTest
+public class CliCommandsErrorTest {
+    private static final String NODE_URL = "http://localhost:10300";
+
+    @Inject
+    private ApplicationContext context;
+
+    private CommandLine cmd;
+    private StringWriter sout;
+    private StringWriter serr;
+    private int exitCode = Integer.MIN_VALUE;
+
+    private void setUp(Class<?> cmdClass) {
+        cmd = new CommandLine(cmdClass, new MicronautFactory(context));
+        sout = new StringWriter();
+        serr = new StringWriter();
+        cmd.setOut(new PrintWriter(sout));
+        cmd.setErr(new PrintWriter(serr));
+    }
+
+    private void execute(Class<?> cmdClass, String urlOptionName, String urlOptionValue, List<String> additionalOptions) {
+        setUp(cmdClass);
+        List<String> options = new ArrayList<>();
+        options.add("--" + urlOptionName + "=" + urlOptionValue);
+        options.addAll(additionalOptions);
+        exitCode = cmd.execute(options.toArray(new String[0]));
+    }
+
+    static List<Arguments> cmdClassAndOptionsProvider() {
+        return List.of(
+                Arguments.arguments(NodeConfigShowSubCommand.class, "node-url", List.of()),
```

Comment:
connect, status, cluster init also have --cluster-url or --node-url options. Shall we add tests for them as well?

Reply:
Agreed, disabled test with the link to jira would be a good solution.

Reply:
> Connect and status commands currently rely on the local installation so they should be tested in the integration tests. ,

I can pass `--cluster-url` option to them, right? Why is local installation required?

Reply:
Ah, sorry, connect command doesn't use the --cluster-url option so some changes in the tests are required to test it.

Reply:
As to why the local installation is required for the status command - because the REST endpoint is not implemented yet?